CODEOWNERS

```diff
@@ -0,0 +1,28 @@
+# CODEOWNERS for autoreview assigning in github
+
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# Root folder
+/CODEOWNERS                               @carlescufi
+/LICENSE                                  @carlescufi
+/Jenkinsfile                              @carlescufi
+/README.rst                               @ru-fu @carlescufi
+
+# All cmake related files
+/CMakeLists.txt                           @SebastianBoe
+
+# All Kconfig related files
+Kconfig*                                  @SebastianBoe
+
+# All documentation related files
+*.rst                                     @b-gent
+
+# All subfolders
+/ble_controller/                          @joerchan @rugeGerritsen
+/bsdlib/                                  @lemrey @evenl
```

Comment:
The code owner of the `bsdlib` in this repository has until now been @rlubos. Unless something has changed we should probably update this entry.

Reply:
I think I agree with @gbakke . Let's add @rlubos here unless he himself has something against it.

Reply:
Added.